Makefile

```
@@ -311,31 +311,37 @@ app_delegate: ## Delegate trust to a gateway (must specify the APP and GATEWAY_A
 
 .PHONY: app1_delegate_gateway1
 app1_delegate_gateway1: ## Delegate trust to gateway1
-	APP=app1 GATEWAY_ADDR=pokt15vzxjqklzjtlz7lahe8z2dfe9nm5vxwwmscne4 make app_delegate
+	GATEWAY1=$(make poktrolld_addr ACC_NAME=gateway1)
+	APP=app1 GATEWAY_ADDR=$(GATEWAY1) make app_delegate
```

Does this  need the double dollar-sign as well?:
```suggestion
	GATEWAY1=$$(make poktrolld_addr ACC_NAME=gateway1)
	APP=app1 GATEWAY_ADDR=$$(GATEWAY1) make app_delegate
```